File: Python/Perro.py
from Mascota import Mascota

class Perro(Mascota):
    def __init__(self, tamanho, nombre, edad):
        super(Perro, self).__init__(nombre, edad)
        self.tamanho = tamanho

    def perseguirLaCola(self):
        return "Me persigo la cola"

    def comunicarse(self):
        return "guau, guau"

    def presentarse(self):
        return super(Perro, self).presentarse() + " perro "

File: Ruby/Perro.rb
require './Mascota'

module Veterinaria
    class Perro < Mascota
        include Veterinaria
        def initialize tamanho, nombre, edad
            @tamanho = tamanho
            super nombre, edad
        end

        def perseguirLaCola
            "Mer persigo la cola"
        end

        def comunicarse
            "guau, guau"
        end

        def presentarse
            super + "perro"
        end
    end
end
